pushapkscript/docker.d/worker.yml

```diff
@@ -100,41 +139,68 @@ products:
           skip_check_same_locales: true
           skip_checks_fennec: true
           apps:
+            fenix-nightly:
+              package_names: [ "org.mozilla.fenix" ]
+              certificate_alias: 'fenix'
```

Comment:
Nit: I wonder if we should call this certificate `fenix-dep`. Having `fenix`, `fenix-release` and all looks confusing. `fenix-dep` would clarify the intent behind this certificate, in my opinion.

Reply:
I hear you! But if I change fenix/focus, I would also want to change reference_browser, which I think has a slightly different configuration, which... I think it's not quite worth the extra effort -- leaving it as-is.